readthedocs/projects/models.py

```diff
@@ -1826,6 +1830,12 @@ class EnvironmentVariable(TimeStampedModel, models.Model):
         on_delete=models.CASCADE,
         help_text=_('Project where this variable will be used'),
     )
+    public = models.BooleanField(
+        _('Public'),
+        default=False,
+        null=True,
```

Comment:
I think it's better if we write a data migration instead (we can remove the form code as well). This way the data will be valid in our DB.